test/node/parser/serializer.test.ts

```diff
@@ -0,0 +1,80 @@
+import * as BSON from '../../register-bson';
+import { expect } from 'chai';
+import { bufferFromHexArray } from '../tools/utils';
+import { ByteUtils } from '../../../src/utils/byte_utils';
+
+describe('serialize()', () => {
+  describe('javascript number', () => {
```

Comment:
Can you clarify why these tests are necessary?

Here are my thoughts.  You did refactor how we serialize numbers, so we need to make sure that the refactor is adequately tested.  But either our existing test coverage is adequate and no reason for new tests here (except for new negative zero tests) or our existing test coverage is inadequate.  If our existing test coverage is inadequate, then we should either revert the refactor in our number parsing so we're not making untested refactors, or add more testing here so our coverage is complete and leave the refactor.

Reply:
they aren't necessary